### ISSUE TITLE
Acessibility - VoiceOver Not Announcing the Footer Icons are Buttons

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#4]
The application does not tell the VoiceOver user that the “All Incidents” and the other three items in the footer can be tapped to go to another screen as a link or that they are a button. I think they should be a link since they are going somewhere and not doing a function.

**Comment by aanthonyDOL** [comment]
Those buttons are tabs; which is how they are defined per the development environment. They essentially are links to another page ; but the technical name is Tab Bar Item.

**Comment by jfc3-dol** [comment]
If they go to another screen they should be a link not a tab.

Well, that could be an issue for users that use assistive technology like a screen reader when they are expecting links and not tabs.

Please let me close the issues that way I can keep track of what has been fixed and reviewed.
